### PR TITLE
Revoke third-party SCEP certs

### PR DESCRIPTION
**Pull request by ChadSimmons**
Added note and reference to call out that third-party SCEP certs are removed but not revoked.

### CODE REVIEW

**Comment by Brenduns** [comment]
@ChadSimmons  Thanks for this PR.  That's a great note to add there.  

#sign-off

File: memdocs/intune/protect/remove-certificates.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ Manual deletion of a certificate is a scenario that applies across platforms and
 
 In this scenario, after the certificate is deleted, the next time the device checks in with Intune it's found to be out of compliance as it is missing the expected certificate. Intune then issues a new certificate to restore the device to compliance. No other action is needed to restore the certificate.
 
+> [!NOTE]
+> SCEP certificates are [removed but not revoked](../certificate-authority-add-scep-overview#removing-certificates) when using a using a third-party certification authority.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/remove-certificates.md:53]:
```suggestion
> SCEP certificates are [removed but not revoked](../certificate-authority-add-scep-overview#removing-certificates) when using a third-party certification authority.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ChadSimmons, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks.

**Pull request merged**